ipinfo/cmd_tool_aggregate.go

```diff
@@ -20,37 +20,40 @@ var completionsToolAggregate = &complete.Command{
 
 func printHelpToolAggregate() {
 	fmt.Printf(
-		`Usage: %s aggregate [<opts>] <cidr | filepath>
+		`Usage: %s tool aggregate [<opts>] <cidr | IP | IP range | filepath>
```

Comment:
See style in other code for listing options in this like. See `git grep 'range |'`.